ui/plugins/annotations_panel.jsx

```diff
@@ -0,0 +1,152 @@
+'use strict';
+const PropTypes = require('prop-types');
+
+// TODO: This is the thRelatedBugSaved directive that's been converted to a component;
+// it should be moved into its own file at some point
```

Comment:
This is actually fine here, imo.  Unless we decide to use it elsewhere at some point.  So we can remove this comment.

Also: You can remove the directive and partial to the original now, since this encompasses everything.  :)